uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/views/CheckoutView.kt

```diff
@@ -372,8 +372,12 @@ internal class CheckoutView @JvmOverloads constructor(context: Context,
                                                  )
         } else {
             bookingRequestPaymentDetailsWidget.setPassengerDetails(passengersDetailLayout.getPassengerDetails())
-            bookingRequestPaymentDetailsWidget.onActivityResult(requestCode, resultCode, data)
+            val paymentSuccess = bookingRequestPaymentDetailsWidget.onActivityResult(requestCode, resultCode, data)
```

Comment:
Maybe we could rename the onActivityResult to processPaymentResult() or something like that

Reply:
the method's also used for countrypicker unfortunatelly